tools/c7n_azure/tests/azure_common.py

```diff
@@ -25,6 +26,9 @@
 
 C7N_SCHEMA = generate()
 DEFAULT_SUBSCRIPTION_ID = 'ea42f556-5106-4743-99b0-c129bfa71a47'
+# latest VCR recording date that tag tests
```

Comment:
Do we know which tests are affected by this? I know I ran into needing to re-record a cassette and then all the tests failed and I had no idea and it ended up being this TEST_DATE. It took me hours to figure it out and it was actually in the file where the tests were breaking. I'm concerned that moving it will make it REALLY difficult for someone to know they need to update this test date unless we comment on the tests themselves.

Reply:
That's true, I spent a lot of time either :)
I added a comment to all tests that might be affected by TEST_DATE.